### PR TITLE
add custom type options

### LINKED ISSUE #27: co-body unsupported content-type: application/graphql
Hello,

cannot post application/graphql as my content type...

Thx

### PR DESCRIPTION
**Pull request by coderhaoxin**
Maybe, it will fix #27

### CODE REVIEW

**Comment by coderhaoxin** [comment]
@fengmk2 any advice?

**Comment by fengmk2** [comment]
LGTM

File: Readme.md [hunk 1]
```diff
@@ -51,6 +54,9 @@ var body = yield parse.text(req);
 
 // either
 var body = yield parse(req);
+
+// custom type
+var body = yield parse(req, { textType: ['text', 'html'] });
```
Comment (fengmk2) [inline Readme.md]: `textTypes` will be better?

**Comment by fengmk2** [comment]
add a test case for `application/graphql` ?

File: Readme.md (hunk 1 above)
Comment (coderhaoxin) [inline Readme.md]: done!

**Comment by coderhaoxin** [comment]
test case for `application/graphql` added

**Comment by fengmk2** [comment]
+1

**Comment by fengmk2** [comment]
3.1.0

**Pull request merged**